eox_nelp/admin/user.py

```diff
@@ -0,0 +1,42 @@
+"""
+This module defines the Django admin configuration for handling user model.
+
+Classes:
+    NelpUserAdmin: Custom admin class for User model to include extra info fields like national_id.
+"""
+from importlib import import_module
+from importlib.util import find_spec
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+
+from eox_nelp.admin.register_admin_model import register_admin_model as register
+
+if find_spec('eox_support') and 'eox_support.apps.EoxSupportConfig' in settings.INSTALLED_APPS:
+    SupportUserAdmin = import_module("eox_support.admin.user").SupportUserAdmin
+else:
+    SupportUserAdmin = BaseUserAdmin
+
+User = get_user_model()
+
+
+class NelpUserAdmin(SupportUserAdmin):
+    """EoxNelp User admin class."""
+    list_display = SupportUserAdmin.list_display[:2] + ('user_national_id',) + SupportUserAdmin.list_display[2:]
+    search_fields = SupportUserAdmin.search_fields + ('extrainfo__national_id',)
+    fieldsets = SupportUserAdmin.fieldsets + (
+        ('Extra info Fields', {'fields': ('user_national_id',)}),
```

Comment:
could you include this in the personal info section ?

Reply:
Why you dont want to split to extra info fields. Here could be first_arabic_name, or is_phone_validated. 
That fields belongs to extra_info model. So they should go to that table or admin model to check

Reply:
This kind of classification just makes sense for us, for someone who doesn't know about tables or models this could be  more confusing, on the other hand, if the purpose is to facilitate  the navigation to extra info table, don't add a national id, add a link to the instance, anyway, if you consider that is too hard or that you have to copy the whole Personal info, to achieve that, please keep this way

Reply:
[Screencast from 05-11-24 11:48:23.webm](https://github.com/user-attachments/assets/6a4f852d-ca19-4941-bff8-5358e5c7aef5)
I like the inline option =)